src/components/ConnectionHelp.js

```diff
@@ -0,0 +1,243 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  IconButton,
+  Dialog,
+  DialogContent,
+  Box,
+  Button,
+  Typography,
+  Link,
+} from '@material-ui/core';
+import {
+  withStyles,
+} from '@material-ui/core/styles';
+import { HelpOutline } from '@material-ui/icons';
+import { FormattedMessage } from 'react-intl';
+import { hot } from 'react-hot-loader';
+
+import RoverConnection from '@/containers/RoverConnection';
+import gigglebotImage from '@/assets/images/connection-help/gigglebot.png';
+import displayImage from '@/assets/images/connection-help/display.png';
+import flashingImage from '@/assets/images/connection-help/flashing.png';
+import deviceWindowsImage from '@/assets/images/connection-help/device-windows.jpg';
+
+// TODO: Use a flag on user model to launch modal immediately
```

Comment:
I went ahead and wired it up to Redux, knowing that soon we'll need to get a boolean from the user model to see if they have "show this every time I log in" checked.